pycqed/measurement/waveform_control/segment.py

```diff
@@ -934,6 +1034,10 @@ def element_start_length(self, element, awg):
             t_start = min(pulse.algorithm_time(), t_start)
             t_end = max(pulse.algorithm_time() + pulse.length, t_end)
 
+        if t_start == float('inf') or t_end == -float('inf'):
```

Comment:
Is there a reason against using np.inf?

Please add a comment explaining in which instances we expect to end up in this branch of code and why it can be silently ignored.

Reply:
Np.inf can be used (I used notation that was used further up in the code, but changed it at both positions. 

Added comment.